cross/alsa-lib/Makefile

```diff
@@ -13,7 +13,7 @@ LICENSE  =
 
 GNU_CONFIGURE = 1
 CONFIGURE_ARGS = --disable-python
-ifeq ($(findstring $(ARCH),88f5281 ppc853x ppc854x),$(ARCH))
+ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
```

Comment:
Missing `include ../../mk/spksrc.cross-cc.mk` before this line.
Can we agree to put the include after GNU_CONFIGURE (or LICENSE, if GNU_CONFIGURE isn't set), and before CONFIGURE_ARGS?
That approach keeps all the CONFIGURE_ARGS more or less in the same place.